Src/Base/AMReX_MultiFab.cpp

```diff
@@ -1586,6 +1587,53 @@ MultiFab::sum (int comp, bool local) const
     return sm;
 }
 
+Real
+MultiFab::sum_unique (int comp,
+                      const Periodicity& period) const
+{
+    BL_PROFILE("MultiFab::sum_unique()");
+
+    using namespace amrex::literals;
+
+    // Owner is the grid with the lowest grid number containing the data
+    std::unique_ptr<iMultiFab> owner_mask = OwnerMask(period);
+
+    Real sm = Real(0.0);
+#ifdef AMREX_USE_GPU
+    if (Gpu::inLaunchRegion()) {
+        auto const& ma = this->const_arrays();
+        auto const& msk = owner_mask->const_array();
```

Comment:
@WeiqunZhang I am not sure how to pass the owner mast onto GPU without a MFI. What am I missing?